plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -777,6 +778,19 @@ public HiveConfig setSortedWritingEnabled(boolean sortedWritingEnabled)
         return this;
     }
 
+    public boolean isOptimizeSortedFileReads()
+    {
+        return optimizeSortedFileReads;
+    }
+
+    @Config("hive.optimize-sorted-file-reads")
+    @ConfigDescription("Enable planner optimizations for reading sorted files")
```

Comment:
remove "planner" -- user should not be concerned which part does the optimization